.github/actions/notify_main_failure/action.yml

```diff
@@ -0,0 +1,44 @@
+name: notify_main_failure
```

Comment:
I don't think this is being used at all in the new GHA workflows? If you want to still send slack notifications, these are the ways you can do this in GitHub Actions - https://github.com/hashicorp/GHA-migration/issues/41

Reply:
@dekimsey just pointed me to this action (https://github.com/hashicorp/actions-slack-status) which should work as a direct replacement for this

Reply:
Thanks! We're removing this in favor of your suggestion